lib/DDG/Spice/CodeSearch.pm

```diff
@@ -6,7 +6,8 @@ use DDG::Spice;
 
 triggers startend => "code", "example";
 
-spice to => 'https://searchcode.com/api/jsonp_codesearch_I/?q=$1&callback={{callback}}';
+spice to => 'https://sourcegraph.com/.api/global-search?Query=$1&Limit=1&Fast=0';
+spice wrap_jsonp_callback => 1;
 
 my $languages = join "|", share('languages.txt')->slurp;
```

Comment:
@jrasanen let's tighten this up. For now, let's switch the trigger to `python`. We can relax the triggering once this is live as necessary